schema/catena.schema.json

```diff
@@ -1162,66 +1204,47 @@
       "default": "UNDEFINED"
     },
     "constraint": {
-        "title": "Constraint to apply to parameter value",
-        "description": "Constraints on how the value of the parameter can change. Its schema is polymorphic dependent on type",
-        "type": "object",
-        "oneOf": [
-          {
-              "properties": {
-                  "type": {
-                      "type": "string",
-                      "enum": [
-                          "UNDEFINED",
-                          "INT_RANGE",
-                          "FLOAT_RANGE",
-                          "INT_CHOICE",
-                          "STRING_CHOICE",
-                          "STRING_STRING_CHOICE",
-                          "ALARM_TABLE"
-                      ],
-                      "default": "UNDEFINED"
-                  }
-              },
-              "required": ["type"],
-              "oneOf": [
-                  {
-                  "$ref": "#/$defs/float32_constraint"
-                  },
-                  {
-                  "$ref": "#/$defs/int32_constraint"
-                  },
-                  {
-                  "$ref": "#/$defs/string_constraint"
-                  }
-              ]
-          },
-          {
-              "$ref": "#/$defs/constraint_ref"
-          }
-      ]
-    },
-    "constraint_ref": {
-        "title": "Constraint Reference",
-        "description": "Reference to a constraint that is shared amongst multiple parameters",
-        "properties": {
-            "ref_oid": {
-                "type": "string",
-                "format": "json-pointer"
-            }
-        },
-        "required": ["ref_oid"],
-        "additionalProperties": false
-    },
-    "constraints": {
+      "title": "Constraint to apply to parameter value",
+      "description": "Constraints on how the value of the parameter can change. Its schema is polymorphic dependent on type",
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": [
+            "UNDEFINED",
+            "INT_RANGE",
+            "FLOAT_RANGE",
+            "INT_CHOICE",
+            "STRING_CHOICE",
+            "STRING_STRING_CHOICE",
+            "ALARM_TABLE"
+          ],
+          "default": "UNDEFINED"
+        }
+      },
+      "required": ["type"],
       "oneOf": [
         {
-          "$ref": "#/$defs/constraint"
+          "$ref": "#/$defs/float32_constraint"
         },
         {
-          "$ref": "#/$defs/oid"
+          "$ref": "#/$defs/int32_constraint"
+        },
+        {
+          "$ref": "#/$defs/string_constraint"
         }
       ]
     },
+    "constraint_ref": {
+      "properties": {
+        "ref_oid": {
+          "type": "string",
```

Comment:
can we add a pattern specifier so that only valid `oids` can be part of the json pointer? https://json-schema.org/understanding-json-schema/reference/string